ios/RNSScreen.m

```diff
@@ -564,6 +564,29 @@ - (void)viewWillAppear:(BOOL)animated
     _shouldNotify = NO;
   }
 
+#if !TARGET_OS_TV
+  NSUInteger currentIndex = [self.navigationController.viewControllers indexOfObject:self];
```

Comment:
I think it should be inside the below `if` since before `iOS` 11 it will be just unused variable.